web/app/themes/ppj/src/sass/breakpoints.scss

```diff
@@ -4,6 +4,10 @@ $breakpoints: (
   'large'     : ( min-width: $length-large )
 );
 
+$vbreakpoints: (
```

Comment:
Please remove `vbp` mixin and associated variables – this is no longer required since vertical breakpoints are no longer used on the site.

- In `breakpoints.scss` remove `$vbreakpoints` variable and `vbp` mixin.
- In `variables.scss` remove `$length-regular` variable.